docs/dictionary/function/machine.lcdoc

```diff
@@ -30,22 +31,24 @@ The <machine> <function> <return|returns> a <string>.
 -   iPhone Simulator: the device is a simulated iPhone
 -   iPad: the device is the iPad
 -   iPad Simulator: the device is a simulator iPad
-
+-   : macOS, Unix and Android returns described below
```

Comment:
Hmm... filemaker will be the same as for the underlying os so I'm not sure it's necessary. It's just for the time being it's the only win 64 bit build we do.